go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/abcxyz/team-link
 
-go 1.22
+go 1.23.4
```

Comment:
Our linter isnt ready for v1.23 yet, this might give you some issues where you'll rabbit hole down that path...

Reply:
Looks like the latest version of the client (v0.117.0) requires go 1.23, so I'll use v0.116.0 instead and we can upgrade later when we're ready.